openslides/core/projector.py

```diff
@@ -13,20 +13,17 @@ class CustomSlideSlide(ProjectorElement):
     Slide definitions for custom slide model.
     """
     name = 'core/customslide'
-    scripts = 'core/customslide_slide.js'
 
     def get_context(self):
         pk = self.config_entry.get('id')
         if not CustomSlide.objects.filter(pk=pk).exists():
             raise ProjectorException(_('Custom slide does not exist.'))
-        return [{
-            'collection': 'core/customslide',
-            'id': pk}]
+        return {'id': pk}
 
     def get_requirements(self, config_entry):
         self.config_entry = config_entry
         try:
-            pk = self.get_context()[0]['id']
+            pk = self.get_context()['id']
```

Comment:
Here we can choose: Do we want that the requirement does nothing if the custom slide does not exist or is it no harm because the client might get a 404 without problems?